spec/models/comfy/cms/page_spec.rb

```diff
@@ -234,6 +234,16 @@
       expect(results.map(&:page_views)).to eq([200, 152, 100])
     end
 
+    it 'should not return articles with zero page_views' do
+      create :page, page_views: 1, position: 1
+      create :page, page_views: 0, position: 2
+      create :page, page_views: 0, position: 3
+
+      results = Comfy::Cms::Page.most_popular(3)
+
+      expect(results.map(&:page_views)).to eq([1])
+    end
+
```

Comment:
@dougdroper I asked because I'm always think in separate contexts when writing the specs. Imho, contexts are a powerful way to make your spec clear and very well organized. And in the long period, contexts keep specs easy to read. Wdyt about it?